packages/inula-adapter/tests/vue/dynamicComponent.test.tsx

```
@@ -0,0 +1,68 @@
+    const container = document.createElement('div');
+    document.body.appendChild(container);
+
+    const CustomComponent = ({ text }) => <div>{text}</div>;
+
+    const app = createApp(<DynamicComponent is="custom-component" text="Hello, Custom Component!" />);
+
+    app.component('CustomComponent', CustomComponent);
+
+    app.mount(container);
+
+    // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)